src/rpc/blockchain.cpp

```diff
@@ -31,6 +31,7 @@
 #include <rpc/server_util.h>
 #include <rpc/util.h>
 #include <script/descriptor.h>
+#include <sqlite3.h>
```

Comment:
I think this makes sqlite3 a required dependency for building Bitcoin Core, right?

Some CI tasks are failing because they don't have `sqlite3.h`.

https://github.com/bitcoin/bitcoin/pull/24952/checks?check_run_id=6139350735

```
rpc/blockchain.cpp:34:10: fatal error: 'sqlite3.h' file not found
#include <sqlite3.h>
         ^~~~~~~~~~~
```

Reply:
Ah, right! Will fix up

Reply:
Forgot to handle the SQLITE_* constants too. Will fix up.